src/config.ts

```diff
@@ -89,6 +89,7 @@ export interface SnowpackConfig {
   exclude: string[];
   knownEntrypoints: string[];
   webDependencies?: {[packageName: string]: string};
+  mapping?: {[ext: string]: 'js' | 'html' | 'css'};
```

Comment:
What are your thoughts on naming this `config.extensionMap` or `config.extMap` instead? Just my opinion, but if I just saw “mapping” in a config, it wouldn’t be immediately clear to me what it’s doing.

Also, thoughts on having this live in `buildOptions` alongside the other build-related stuff?

Reply:
Sure, I wasn't happy with the name anyways, lets go with `config.extensionMap`. I'll make an update.

I'd put it in `config.buildOptions`, but it's also needed during the `dev` command (so it seems like a top level thing to me). That said, would you prefer it there? It's water under the bridge to me :smile: